beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/model/EntityHousekeepingTable.java

```diff
@@ -0,0 +1,290 @@
+/**
+ * Copyright (C) 2019-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.beekeeper.core.model;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.UpdateTimestamp;
+
+import com.expediagroup.beekeeper.core.error.BeekeeperException;
+import com.expediagroup.beekeeper.core.monitoring.MetricTag;
+
+@Entity
+@Table(name = "table_housekeeping")
+public class EntityHousekeepingTable implements Housekeeping {
```

Comment:
I think maybe we can call this EntityHousekeepingMetadata
And add a column: partitionValue String, which will be null if it is the table? thoughts?

Reply:
Agree on not having the name "table" in it. I think we can also remove `Entity`, we only did that for the previous class to not clash with the `HousekeepingPath` interface name. `HouseKeepingMetadata` sounds good?

Reply:
I will add the partition value in the next set of changes. I didn't want to add the partition stuff in this PR because it was already getting too big. Is that okay?